# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/web-locks/signal.tentative.https.any.js.ini
# Commit: 9e364e2eda05
# Full Hash: 9e364e2eda05d2b22080a79ce291cd954946bfc3
# Author: Kagami Sascha Rosylight <krosylight@mozilla.com>
# Date: 2023-02-24 16:04:01
# Regressor Bug: 1818468
# File Overlap Count: 1
# Description:
#   Bug 1818468 - Throw `signal.reason` when already aborted in `navigator.locks.request` r=evilpie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D170742
# ==============================================================================

diff -r 398115c86eed -r 9e364e2eda05 testing/web-platform/meta/web-locks/signal.tentative.https.any.js.ini
--- a/testing/web-platform/meta/web-locks/signal.tentative.https.any.js.ini	Fri Feb 24 12:11:28 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,38 +0,0 @@
-[signal.tentative.https.any.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [Passing an already aborted signal rejects with the custom abort reason.]
-    expected: FAIL
-
-  [Passing an already aborted signal rejects with the default abort reason.]
-    expected: FAIL
-
-
-[signal.tentative.https.any.serviceworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [Passing an already aborted signal rejects with the custom abort reason.]
-    expected: FAIL
-
-  [Passing an already aborted signal rejects with the default abort reason.]
-    expected: FAIL
-
-
-[signal.tentative.https.any.sharedworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [Passing an already aborted signal rejects with the custom abort reason.]
-    expected: FAIL
-
-  [Passing an already aborted signal rejects with the default abort reason.]
-    expected: FAIL
-
-
-[signal.tentative.https.any.worker.html]
-  expected:
-    if (os == "android") and fission: [TIMEOUT, OK]
-  [Passing an already aborted signal rejects with the custom abort reason.]
-    expected: FAIL
-
-  [Passing an already aborted signal rejects with the default abort reason.]
-    expected: FAIL
